### DISCUSSION TITLE
A couple of comments on the new Zettlr v.2

### DISCUSSION THREAD

**Discussion by tomislavmedak** [Zettlr/Zettlr#2826] [General]
Exciting to see a new milestone and the amount of development that you put in into Zettlr! A couple of quick comments on the editor after giving the new Zettlr a quick spin:

It's entirely unexpected behaviour that hitting the zoom-in increases not only the font size in the editor, but the entire application -- including icons, menues and window decorations. Probably related, toggling-in the file manager extends the width of the text column. 

In the dark theme mode, the white color of the font is too much of a contrast. For easier reading, the default should be a light gray font, such as, for instance, in the dark theme on Atom.

Selected folder in the file manager is highlighted also with too much contrast. Maybe, instead of using the primary theme color, using another hue of light/dark grey, depending on the mode, might be less distracting and easier on the eyes.

Headings are way too big. You want to be able to know where you are, but maybe using a different color might do the trick instead of the size and bold-type.

Underlines to suggest spelling errors could be a bit thinner and maybe a couple of pixel-points under the word.

**Comment by nathanlesage** [comment]
Thanks for the suggestions! Feel free to open these as separate issues so I can tend to them; I can't really keep up with discussions here when they contain specific things to change :)
